cmd/asset-syncer/postgresql_utils.go

```diff
@@ -134,15 +152,15 @@ func (m *postgresAssetManager) removeMissingCharts(charts []models.Chart) error
 		chartIDs = append(chartIDs, fmt.Sprintf("'%s'", chart.ID))
 	}
 	chartIDsString := strings.Join(chartIDs, ", ")
-	rows, err := m.DB.Query(fmt.Sprintf("DELETE FROM %s WHERE info ->> 'ID' NOT IN (%s)", chartTable, chartIDsString))
+	rows, err := m.DB.Query(fmt.Sprintf("DELETE FROM %s WHERE info ->> 'ID' NOT IN (%s) AND info -> 'repo' ->> 'name' = $1", dbutils.ChartTable, chartIDsString), charts[0].Repo.Name)
```

Comment:
Had to add the `AND ...` clause because it was deleting charts from other repositories :sweat_smile:

Reply:
Heh - good to catch that one early :P